src/bp/core/services/module/resources-loader.ts

```diff
@@ -147,4 +168,27 @@ export class ModuleResourceLoader {
       .global()
       .upsertFile('/', filename, `${CHECKSUM}${this._calculateHash(fileContent)}${os.EOL}${fileContent}`)
   }
+
+  @WrapErrorsWith(args => `Error in migration script "${args[2]}" located at "${args[3]}".`)
+  private async _executeMigration(migrationsFile: string) {
+    const content: ModuleMigrationInstruction[] = JSON.parse(fse.readFileSync(migrationsFile, 'utf8'))
+    if (!content) {
+      throw new Error(`Expected a valid JSON object.`)
+    }
+
+    for (const migration of content) {
+      if (!Array.isArray(migration.filesToDelete)) {
+        continue
+      }
+
+      for (const fileToDelete of migration.filesToDelete) {
```

Comment:
What happens if the file was changed manually ?

Reply:
we delete it nonetheless for now since it's said to be deleted